src/sha256.hpp

```diff
@@ -15,31 +15,31 @@ class Sha256 {
   using hash_t = vector<uint8_t>;
 
 public:
-  inline static bool isMatch(string_view target_message, const hash_t &hashed_message) {
+  inline static bool isMatch(const string_view &target_message, const hash_t &hashed_message) {
     auto hashed_target_message = hash(target_message);
     bool result = hashed_target_message == hashed_message;
 
     return result;
   }
 
-  inline static hash_t hash(string_view message) {
+  inline static hash_t hash(const string_view &message) {
     auto msg_bytes = TypeConverter::stringToBytes(message);
     return hash(msg_bytes);
   }
 
-  static hash_t hash(vector<uint8_t> &&msg_bytes) {
+  static hash_t hash(const vector<uint8_t> &&msg_bytes) {
```

Comment:
이 함수와 밑에 l-ref를 받는 hash를 두개 만드는 것보다 universal reference로 만드는게 좋을 것 같습니다.


```suggestion
 template <typename T>
 static hash_t hash(const vector<uint8_t> &&msg_bytes) {
  // main logic
 }
```

Reply:
반영하였습니다.